contracts/prize-pool/yield-source/YieldSourcePrizePool.sol

```diff
@@ -31,7 +34,7 @@ contract YieldSourcePrizePool is PrizePool {
     public
     initializer
   {
-    require(address(_yieldSource) != address(0), "YieldSourcePrizePool/yield-source-zero");
+    require(address(_yieldSource).isContract(), "YieldSourcePrizePool/yield-address-invalid");
```

Comment:
Would make more sense if it was `YieldSourcePrizePool/yield-source-address-invalid` or maybe `YieldSourcePrizePool/yield-source-not-contract-address`